utils/rpms/daos.spec

```
@@ -240,6 +248,7 @@ install -m 644 utils/systemd/%{server_svc_name} %{?buildroot}/%{_unitdir}
 install -m 644 utils/systemd/%{agent_svc_name} %{?buildroot}/%{_unitdir}
 mkdir -p %{?buildroot}/%{conf_dir}/certs/clients
 mv %{?buildroot}/%{_prefix}/etc/bash_completion.d %{?buildroot}/%{_sysconfdir}
+install -m src/client/java %{_prefix}/lib/daos/TESTING/ftest/java/
```

I'm not sure there is actually anything _Functional testing_ specific about this.  Probably more generically:
```suggestion
install -m src/client/java %{_prefix}/lib/daos/TESTING/java/
```
is appropriate.